lighthouse-core/test/module/index.js

```diff
@@ -128,4 +128,24 @@ describe('Module Tests', function() {
     const lighthouseModule = require('../..');
     assert.ok(Array.isArray(lighthouseModule.getAuditList()));
   });
+
+  it('should return a list of audits', function() {
+    const lighthouseModule = require('../..');
+    const lighthouseTraceCategories = lighthouseModule.traceCategories;
+    const requiredTraceCategories = [
```

Comment:
I don't think we want to test the exact contents of the trace categories, as any future updates to the list will have to edit it in both places. Maybe just check that it's an array (as you are below) and that it's non-empty?